packages/renderer/src/lib/dialogs/QuickPickInput.svelte

```diff
@@ -122,6 +122,7 @@ async function onInputChange(event: any) {
   validationError = undefined;
   const value = event.target.value;
   const result = await window.sendShowInputBoxValidate(currentId, value);
+  // eslint-disable-next-line no-null/no-null
   if (result !== undefined && result !== null && result) {
```

Comment:
isn't `if(result)` enough?

Reply:
yes it is

I'll review all if clauses

For now I didn't change them.

Reply:
updated